.github/workflows/ci.yml

```diff
@@ -29,7 +29,9 @@ jobs:
       - name: Set up Terraform
         uses: hashicorp/setup-terraform@v2
         with:
-          terraform_wrapper: false          
+          terraform_wrapper: false
```

Comment:
pins should be added to action as well, re future proofing against such behavior.

Reply:
deferring this to a separate pr, as its not specific to tf